src/type/redux-action.type.ts

```diff
@@ -5,9 +5,16 @@ export interface IPayload<T> {
   payload: AxiosPromise<T>;
   meta?: any;
 }
-export type IPayloadResult<T> = ((dispatch: any) => IPayload<T> | Promise<IPayload<T>>);
+export type IPayloadResult<T> =
+  | ((dispatch: any) => IPayload<T> | Promise<IPayload<T>>)
+  | ((dispatch: any, getState: any) => IPayload<T> | Promise<IPayload<T>>);
```

Comment:
Did you try `export type IPayloadResult<T> = ((dispatch: any, getState?: any) => IPayload<T> | Promise<IPayload<T>>);`

Reply:
I didn't thought of it !